NPE in AboutActivity

### Describe your problem!

From Google Play console, happening in 2021.11.21 (but also in older versions):

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3520)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3692)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2142)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:236)
  at android.app.ActivityThread.main (ActivityThread.java:7864)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:620)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1011)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.AboutActivity$SystemViewCreator.setContent (AboutActivity.java:260)
  at cgeo.geocaching.activity.TabbedViewPagerFragment.notifyDataSetChanged (TabbedViewPagerFragment.java:61)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.setOrderedPages (TabbedViewPagerActivity.java:110)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.createViewPager (TabbedViewPagerActivity.java:49)
  at cgeo.geocaching.AboutActivity.onCreate (AboutActivity.java:102)
  at android.app.Activity.performCreate (Activity.java:8093)
  at android.app.Activity.performCreate (Activity.java:8067)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1309)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3493)
```

Occurrences: 305 by 299 users
Android versions 9-12
Various devices 

### How to reproduce?

Unclear

### Actual result after these steps?

_No response_

### Expected result after these steps?

_No response_

### Reproducible

Unclear

### c:geo Version

2021.11.21

### System information

_No response_

### Additional Information

_No response_

A similar stack trace but listed separate on Google Play:

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3520)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3692)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2142)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:236)
  at android.app.ActivityThread.main (ActivityThread.java:7864)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:620)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1011)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.AboutActivity$SystemViewCreator.setContent (AboutActivity.java:260)
  at cgeo.geocaching.activity.TabbedViewPagerFragment.notifyDataSetChanged (TabbedViewPagerFragment.java:61)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.setOrderedPages (TabbedViewPagerActivity.java:110)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.createViewPager (TabbedViewPagerActivity.java:49)
  at cgeo.geocaching.AboutActivity.onCreate (AboutActivity.java:102)
  at android.app.Activity.performCreate (Activity.java:8093)
  at android.app.Activity.performCreate (Activity.java:8067)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1309)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3493)
```

--- a/main/src/cgeo/geocaching/AboutActivity.java
+++ b/main/src/cgeo/geocaching/AboutActivity.java
@@ -160,7 +160,7 @@ public class AboutActivity extends TabbedViewPagerActivity {
         @Override
         public void setContent() {
             final AboutActivity activity = (AboutActivity) getActivity();
-            if (activity == null) {
+            if (activity == null || binding == null) {
                 return;
             }
             binding.getRoot().setVisibility(View.VISIBLE);
@@ -254,7 +254,7 @@ public class AboutActivity extends TabbedViewPagerActivity {
         @Override
         public void setContent() {
             final AboutActivity activity = (AboutActivity) getActivity();
-            if (activity == null) {
+            if (activity == null || binding == null) {
                 return;
             }
             binding.getRoot().setVisibility(View.VISIBLE);
@@ -296,6 +296,10 @@ public class AboutActivity extends TabbedViewPagerActivity {
 
         @Override
         public void setContent() {
+            if (binding == null || getActivity() == null) {
+                return;
+            }
+
             binding.getRoot().setVisibility(View.VISIBLE);
             setClickListener(binding.license, "https://www.apache.org/licenses/LICENSE-2.0.html");
             final Markwon markwon = Markwon.create(getActivity());
@@ -334,7 +338,7 @@ public class AboutActivity extends TabbedViewPagerActivity {
         @Override
         public void setContent() {
             final Activity activity = getActivity();
-            if (activity == null) {
+            if (activity == null || binding == null) {
                 return;
             }
             binding.getRoot().setVisibility(View.VISIBLE);

